libs/tup-components/src/tickets/TicketCreateModal/TicketCreateModal.tsx

```diff
@@ -18,8 +20,8 @@ const TicketCreateModal: React.FC = () => {
 
   return (
     <>
-      <Button type="secondary" onClick={() => toggle()}>
-        + New Ticket
+      <Button type="link" onClick={() => toggle()}>
```

Comment:
Couple of notes: 
- This overwrites the previous UI of having a normal button for the "New Ticket" modal in the dashboard. We should pass a `display` prop of type `'button' | 'link'` to let the container component decide how it wants to render the button.
-  Instead of a `title` prop, let's pass `children` so the markup looks like ```<TicketCreateModal display='link'>Get Help</TicketCreateModal>```

Reply:
<details><summary>I think there is more to consider about the first note (the new prop).</summary>

I like the `display` prop idea. [Another PR has a similar use case.](https://github.com/TACC/tup-ui/pull/139#discussion_r1103122280) If we go this route, I propose the prop name `display-as` or `as`. An `href` prop would also be necessary, right? At which point the `<Button>` ceases to be a "button" and becomes an "action" (`<Action>`) with a less coupled-to-markup API.

I am more tempted to see a `<A>` or in-house `<Link>` component, then can be told to look like a button. Because [the use of <button> and `<a>` is not arbitrary](https://confluence.tacc.utexas.edu/x/jgXiBw), so I suggest the usage be abstracted/controlled (e.g. `<Action>` proposal in first paragraph) **or** taught via distinct components.

<sup>P.S. The attribute [`is`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/is) is standard global attribute with a reverse meaning as my proposed prop `as`.</sup>

</details>

Reply:
I'm not sure I follow. This component is always going to be a button, we just need to control which prop we pass to the core-components `<Button/>`. There shouldn't be any use for an anchor/href because no navigation is involved.

Reply:
Great. I don't want to go that far either. And, based on your response, I think I now better understand "`display` prop of `button | type`". Your proposal is

- replace: `<Button type="link"...>`
- with: `<Button display="button" /* or "link" */ type="secondary"...>`

Am I correct?

Reply:
[FP-1972](https://jira.tacc.utexas.edu/browse/FP-1972) Button has Display Prop (Instead of type="link")